lib/jekyll/cloudinary.rb

```diff
@@ -218,41 +217,19 @@ def render(context)
         end
 
         # Get source image natural width
-        image = false
-        fallback_url = "https://res.cloudinary.com/#{settings["cloud_name"]}/image/#{type}/#{transformations_string}w_#{preset["fallback_max_width"]}/#{image_url}"
-        natural_width = 100_000
-        width_height = ""
-        # Standard fetch
-        if type == "fetch"
-          # Trap format support issues from ImageMagick
-          begin
-            if is_image_remote
-              image = Magick::ImageList.new
-              urlimage = open(image_url) # Image Remote URL 
-              image.from_blob(urlimage.read)
-            elsif File.exist?(image_path)
-              image = Magick::Image::read(image_path).first
-            end
-          rescue
-            image = false
-          end
-          if image
-            natural_width = image.columns
-            natural_height = image.rows
-            width_height = "width=\"#{natural_width}\" height=\"#{natural_height}\""
-          else
-            fallback_url = image_url
-            Jekyll.logger.warn(
-              "[Cloudinary]",
-              "Couldn't find this image to check its width: #{image_path}. \
-              Try to run Jekyll build a second time. If it still doesn’t work \
-              it could be that your ImageMagick install doesn’t support this format."
-            )
-          end
-        end
-
-        if preset["width_height"] == false
```

Comment:
I ended up taking it out as it was no longer necessary. It was all working around ImageMagick’s treatment of local and remote images. FastImage doesn’t care. Are you seeing an issue?